cmd/esc/cli/testdata/env-set-secret.yaml

```diff
@@ -100,3 +178,13 @@ stderr: |
   > esc env get test password --show-secrets
   > esc env set test password YMHdog6VkUqbVS5dAErxWeMy83r4HeBeNRdEuUFwK3M= --plaintext
   > esc env get test
+  > esc env set test password 1234 --secret
+  > esc env get test
+  > esc env set test password 123.4 --secret
+  > esc env get test
+  > esc env set test password false --secret
+  > esc env get test
+  > esc env set test password true --secret
+  > esc env get test
+  > esc env set test password [] --secret
+  test:3:21: secret values must be string literals
```

Comment:
Is this error message still accurate?

Reply:
Roughly, yes. If you're editing the YAML document directly, secret values must be string literals. On the CLI we allow you to fudge and present non-string scalars.